gr-blocks/python/blocks/qa_file_source_sink.py

```diff
@@ -104,6 +105,67 @@ def test_file_source_can_seek_after_open(self):
 
         os.remove(filename)
 
+    def test_begin_tag(self):
+        src_data = range(1000)
+        expected_result = range(1000)
+
+        filename = "tmp.32f"
```

Comment:
Good idea. I just did it the same way as for the other tests in qa_file_source_sink.py. Do you mean I should change it for the "old" tests as well ? Would it not be outside the scope of the pull request then ?